FreeRTOS/FreeRTOS-Partner-Supported-Demos
# Adding demos for Infineon AURIX TC375 using AURIX Development Studio

Adding demos for Infineon AURIX TC375 using AURIX Development Studio

Description
-----------
FreeRTOS demos for the Infineon AURIX TC3xx FreeRTOS Kernel Port.

Test Steps
-----------
Setup, testing and debugging steps are described in detail in  `AURIX_TC375_ADS/README.md`

Related Issue
-----------
FreeRTOS/FreeRTOS-Kernel-Partner-Supported-Ports/pull/11


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
